### ISSUE TITLE
Does not work on multiple Sidekiq pods

### ISSUE THREAD

**Issue by linyaoli** [arturictus/sidekiq_alive#9]
Let's say i have several sidekiq pods, each is working on a specific queue:
```
backend-staging-sidekiq-alice-stable-96186k9mq6   1/1       Running   2          36m
backend-staging-sidekiq-bob-stable-39797474-8j15c         1/1       Running   2          36m
backend-staging-sidekiq-chris-3103080732-hrsq9              1/1       Running   3          36m
```
Alice pod -> `queue_a`
Bob pod -> `queue_b`
Chris pod -> `queue_c` and `default`

because `SidekiqAlive::Worker` are all enqueued into `default`, it will only be executed by Chris.

But here in
https://github.com/arturictus/sidekiq_alive/blob/2519841d0fe1e7587cf9dfc7bac6325a45a8a7c3/lib/sidekiq_alive/worker.rb#L6-L16

as for the condition:
`if current_hostname == hostname`
if a `SidekiqAlive::Worker` was created by Alice, this condition will always be false, 
because
`current_hostname` is `backend-staging-sidekiq-christ-3103080732-hrsq9` but `hostname` is `backend-staging-sidekiq-alice-stable-96186k9mq6`, thus it will instantly enqueue another job with the alice hostname. This cycle goes indefinitely.

**Comment by gsmetal** [comment]
+1 for this issue. 
Workarounded it by configuring queue to current sidekiq's queue:
```ruby
Sidekiq.configure_server do |config|
  SidekiqAlive::Worker.sidekiq_options queue: config.options[:queues].first  # we should use queue of this sidekiq
end
```

**Comment by arturictus** [comment]
@linyaoli #13 fixes that.
The best approach is to add a dedicated queue`sidekiq_alive` for all your pods. Anyway it will try to find a queue with the @gsmetal approach. Thanks @gsmetal.

https://github.com/arturictus/sidekiq_alive/blob/master/lib/sidekiq_alive.rb#L33-L39

This pull requests is included in release 1.1.0

**Comment by linyaoli** [comment]
Good to know, thanks for the update!
